### ISSUE TITLE
Arbitrary number of reloads before stopping (

### ISSUE THREAD

**Issue by lightinwater** [pylorak/reloadmatic#15]
Would be useful for grindy browser games with "action point" limits :)

**Comment by pylorak** [comment]
Do you mean an option that instead of refreshing a page infinitely often (like now) refreshes only at most X times?

**Comment by lightinwater** [comment]
Correct. A UI element that lets you choose the number of reloads before stopping

**Comment by Kkloe** [comment]
This would be nice, I was just going to suggest to be able to set reload between some hours, like 12:00 to 14:00, but this would suffice
